forwards_paper.tex

```diff
@@ -137,10 +138,15 @@ \section*{Introduction}
 of the \emph{sampled} individuals,
 which results in considerable savings.
 Another way of representing this same information
-is known as the \emph{ancestral recombination graph}, or {ARG} \citep{griffiths1997ancestral},
+is known as the \emph{ancestral recombination graph} ---
+the {ARG} \citep{griffiths1997ancestral},
+where nodes are common ancestor or recombination events ---
 which has been the subject of substantial study
 under the assumptions of coalescent theory \citep{wiuf1997number,wiuf1999ancestry,marjoram2006coalescent,wilton2015smc}.
+\jda{would be nice to say, concisely the difference}
```

Comment:
between what and what?  I was trying not to get into this since 'ARG' means diff things to diff people.

Reply:
I added something later, which I think you incorporated